src/wh_client_cryptocb.c

```diff
@@ -594,6 +585,57 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
         }
         break;
 #endif /* !WC_NO_RNG */
+#ifdef WOLFSSL_CMAC
+    case WC_ALGO_TYPE_CMAC:
+        /* in, key and out are after the fixed size fields */
+        in = (uint8_t*)(&packet->cmacReq + 1);
+        key = in + info->cmac.inSz;
+        out = (uint8_t*)(&packet->cmacRes + 1);
```

Comment:
Input parameter validations are required here since we use them to compute pointers.